### ISSUE TITLE
lensfun-update-data breaks added adapters

### ISSUE THREAD

**Issue by Computerfreaked** [lensfun/lensfun#593]
Using lensfun 0.3.95.

I had added adapters using lensfun-add-adapter and all is ok. Ran lensfun-update-data, update successfully performed and new data in /home/myusername/.local/share/lensfun.

lensfun-update-data Creates a directory "updates/version_2" with the new data (works). But (at least using Darktable) the /usr/local/share/lensfun/version_2/_mounts.xml file is ignored so adapted lenses are no longer available.

Running lensfun-add-adapter again and adding the adapters again creates a file /home/myusername/.local/share/lensfun/_mounts.xml. This is apparently in the wrong location since moving the file to /home/myusername/.local/share/lensfun/updates/version_2 fixes the problem.

Long story short, either do not ignore the standard file /usr/local/share/lensfun/version_2/_mounts.xml or have users rerun lensfun-add-adapter but then the _mounts.xml has to be put in the proper location by lensun-add-adapter.

**Comment by uchrisu** [comment]
I got the same problem right now, using my Sigma 70-200 Sports with MC-11 on Sony E.

In this special case I would add an "Sony E" mount to this specific lens, since Sigma themselves claimes this as compatible to Sony E with the own adapter. But still that is not a solution for all users and neither for all of my adapted lenses.

Is there any way to get it work with lensfun 0.3.95 as most users will use this for now and probably for still a long time?

**Comment by uchrisu** [comment]
I added a fix for this issue. 
I tested the behavior on my machine and it works fine. I think it is in general a good idea to give priority to the user-added profiles. 

Can you please have a look on it an merge it @bronger  ?
